### PR TITLE
Update communitytools.js

### PR DESCRIPTION
**Pull request by blueffect**
New Helium Service Add Request

### CODE REVIEW

**Comment by danielcolinjames** [comment]
hey @blueffect, thanks for submitting this PR!

just wanted to clarify something before we add this to Explorer: I'm curious if you're using the public Helium API for this, or if you have your own ETL/API set up?

just wondering because it sounds like a service that might be polling the API quite frequently, which could be a really big strain on the public API (api.helium.io) depending on how much caching is in place

just wanted to make you aware of that possible/likely limitation in case this uses the public API

**Comment by blueffect** [comment]
Thank you for the information @danielcolinjames. Helium Board has it's own ETL and except a cron which runs rarely, it does not use Helium API at all. System totally runs on it's own database and Blockchain.

File: data/communitytools.js [hunk 1]
```diff
@@ -216,6 +216,12 @@ export const communityToolsList = [
     url: 'https://hs-analyzer.com',
     description: 'HS Analyzer is simple tool for checking your Helium hotspot and its stats (RSSI, Valid / Invalid Beacons etc.).',
     tags: [types.monitoring]
+  },
+  {
+    name: 'Helium Board',
+    url: 'https://www.heliumboard.com/',
+    description: 'Helium Board is sophisticated tool for seeing reports for Helium hotspots, overall Helium network, receiving alerts for set triggers and access multiple tools (PeerPing, PeerBook Inquiry, Hotspot Analysis etc.)',
+    tags: [types.monitoring] [types.planning]
```
Comment (danielcolinjames) [inline data/communitytools.js]:
```suggestion
    tags: [types.monitoring, types.planning],
```

File: data/communitytools.js [hunk 2]
```diff
@@ -216,6 +216,12 @@ export const communityToolsList = [
     url: 'https://hs-analyzer.com',
     description: 'HS Analyzer is simple tool for checking your Helium hotspot and its stats (RSSI, Valid / Invalid Beacons etc.).',
     tags: [types.monitoring]
+  },
+  {
+    name: 'Helium Board',
+    url: 'https://www.heliumboard.com/',
+    description: 'Helium Board is sophisticated tool for seeing reports for Helium hotspots, overall Helium network, receiving alerts for set triggers and access multiple tools (PeerPing, PeerBook Inquiry, Hotspot Analysis etc.)',
```
Comment (danielcolinjames) [inline data/communitytools.js]:
```suggestion
    description: 'Helium Board is a sophisticated tool for seeing reports for Helium hotspots and the overall Helium network, as well as receiving alerts for set triggers, and accessing multiple tools (PeerPing, PeerBook Inquiry, Hotspot Analysis etc.)',
```

**Review by danielcolinjames — APPROVED** [review]

**Comment by danielcolinjames** [comment]
meant to write a reply with my review :) thanks for confirming @blueffect, that's great! going to merge this now

**Pull request merged**